apps/console/src/components/pages/protected/tasks/create-task/sidebar/task-details-sheet.tsx

```diff
@@ -347,14 +380,14 @@ const TaskDetailsSheet = () => {
                             />
                           </div>
                           <FormControl>
-                            <Textarea rows={7} id="details" {...field} className="w-full" />
+                            <PlateEditor onChange={handleDetailsChange} initialValue={taskData?.details ?? undefined} />
                           </FormControl>
                           {form.formState.errors.details && <p className="text-red-500 text-sm">{form.formState.errors.details.message}</p>}
                         </FormItem>
                       )}
                     />
                   ) : (
-                    taskData?.details
+                    <div>{taskData?.details ? helper.convertToReadOnly(taskData.details) : ''}</div>
```

Comment:
```suggestion
<>
{!!taskData?.details  && <div>{helper.convertToReadOnly(taskData.details)}</div> }
</>
```
no need for empty div I assume?